examples/lighting-app/bouffalolab/bl602/README.md

```diff
@@ -1,90 +1,95 @@
-#CHIP BL602 Lighting Example
+CHIP BL602 Lighting Example
+
+## Supported Devices
+
+The CHIP demo application is intended to work on BL602.See
+[Demo board](https://dev.bouffalolab.com/product)
 
 ## Build
 
--   Install some additional tools(likely already present for CHIP developers):
+-   Setting up the environment
+
+```
+$ sudo apt-get update
+$ sudo apt-get upgrade
+$ sudo apt-get install git gcc g++ python pkg-config libssl-dev libdbus-1-dev libglib2.0-dev libavahi-client-dev ninja build python3-venv python3-dev python3-pip unzip libgirepository1.0-dev libcairo2-dev bluez avahi-daemon
+$ sudo apt-get install pi-bluetooth (if not raspberry pi sudo apt-get install bluetooth)
+$ reboot
+
+```
+
+-   Install packets
 
-    #Linux \$ sudo apt-get install git libwebkitgtk-1.0-0 ninja-build
+```
+$ cd {path-to-connectedhomeip}
+connectedhomeip$ source ./scripts/bootstrap.sh
+connectedhomeip$ source ./scripts/activate.sh
 
-    #Mac OS X \$ brew install ninja
+```
 
-*   Build the example application:
+-   Build the example application:
 
-    `connectedhomeip$ ./scripts/build/build_examples.py --target bl602-lighting build`
+    `connectedhomeip$ ./scripts/build/build_examples.py --target bl602-light build`
 
 Generated files
 
-         ```
-         connectedhomeip/out/lighting_app_bl602/chip-bl602-lighting-example.bin
-         ```
+         connectedhomeip/out/bl602-light/chip-bl602-lighting-example.bin
 
 -   To delete generated executable, libraries and object files use:
 
-         ```
-         $ cd ~/connectedhomeip/
-         $ rm -rf out/
-         ```
+          $ cd ~/connectedhomeip/
```

Comment:
backtics seemed correct for these (and probably the items above as well) as these are code not text.

Reply:
Thanks for pointing it out. I changed it